### DISCUSSION TITLE
/articles/sharepoint-framework-definitive-guide-sharepoint-server-subscription-edition/

### DISCUSSION THREAD

**Discussion by giscus** [Voitanos/voitanos.io-comments#242] [Comments]
These days, all of Microsoft's updates about the SPFx, or community content have one thing in common: they ignore SharePoint Server SE. Not this post!
https://www.andrewconnell.com/articles/sharepoint-framework-definitive-guide-sharepoint-server-subscription-edition/

**Comment by dstretch** [comment]
This is really valuable information, thank you!

Sadly since my company requires us to use versions of software that are actually still in support it basically means I can't develop SPFx web parts and stay in compliance. Come on, Microsoft 🤬

NodeJS v8 is 7 years old and Node say they provide critical fixes for 30 months on average so this is an instant no-go for me 😭

**Reply by andrewconnell** [reply]
I suggest you share that feedback with Microsoft... they need to hear this type of feedback straight from affected customers: https://github.com/SharePoint/sp-dev-docs/issues
